2D/02_Inference.ipynb

```diff
@@ -28,22 +28,95 @@
    "metadata": {},
```

Comment:
When I try to view the notebook in GitHub, I get the error: Invalid Notebook. Could you check @asirvaiy ..

Reply:
Hi @ravi9 Is this issue only for this notebook or all the notebooks(updated)?

Reply:
The issue is for inference-solution.ipynb notebook due to missing ending syntax. Fixing it.

Reply:
Now inference-solution.ipynb should open @ravi9 please check.